packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -85,24 +85,47 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
         return onSuccess?.();
       })
       .catch(err => {
-        if (isClerkAPIResponseError(err)) {
-          removeInvalidEmails(err.errors[0]);
+        if (!isClerkAPIResponseError(err)) {
+          handleError(err, [], card.setError);
+          return;
         }
 
-        if (isClerkAPIResponseError(err) && err.errors?.[0]?.code === 'duplicate_record') {
-          const unlocalizedEmailsList = err.errors[0].meta?.emailAddresses || [];
-          card.setError(
-            t(
-              localizationKeys('organizationProfile.invitePage.detailsTitle__inviteFailed', {
-                // Create a localized list of email addresses
-                email_addresses: createListFormat(unlocalizedEmailsList, locale),
-              }),
-            ),
-          );
-        } else if (isClerkAPIResponseError(err) && err.errors?.[0]?.code === 'form_param_format_invalid') {
-          card.setError(translateError(err.errors[0]));
-        } else {
-          handleError(err, [], card.setError);
+        removeInvalidEmails(err.errors[0]);
+
+        switch (err.errors?.[0]?.code) {
+          case 'duplicate_record': {
+            const unlocalizedEmailsList = err.errors[0].meta?.emailAddresses || [];
+            card.setError(
+              t(
+                localizationKeys('organizationProfile.invitePage.detailsTitle__inviteFailed', {
+                  // Create a localized list of email addresses
+                  email_addresses: createListFormat(unlocalizedEmailsList, locale),
+                }),
+              ),
+            );
+            break;
+          }
+          case 'already_a_member_in_organization': {
+            const longMessage = err.errors[0].longMessage ?? '';
+            const email = longMessage.match(/\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b/)?.[0];
+
+            if (!email) {
```

Comment:
I think this is impossible case unless FAPI changes the error message to not include an email, otherwise we'll fallback to their default `long_message` instead of translating.